src/server/pfs/pfs.go

```diff
@@ -121,6 +127,10 @@ func (e ErrCommitNotFinished) Error() string {
 	return fmt.Sprintf("commit %v not finished", e.Commit.ID)
 }
 
+func (e ErrAmbiguousCommit) Error() string {
+	return fmt.Sprintf("commit %v is ambiguous (specify the commit branch to resolve)", e.Commit.ID)
```

Comment:
```suggestion
	return fmt.Sprintf("commit %v is ambiguous (specify the branch to resolve)", e.Commit.ID)
```
I think the word "commit" is just going to confuse users since the thing they need to specify is just, for example, "master" right which they just thing of as a branch.